best-practices/new_project_checklist.md

```diff
@@ -0,0 +1,78 @@
+# New Project "To Production" Checklist
+
+Feel free to make a copy of this checklist when beginning work on a new development project. You may or may not wish to store the copy for posterity, e.g., as part of the [project's DevOpsDocs](https://github.com/sul-dlss/DevOpsDocs).
```

Comment:
While I'm not 👎 for the changes here, I am curious if we do have a place for accounting for how each production service does or does not adhere and iterate on this checklist. I agree that having it in DevDopsDocs might have not made sense for a first responder scenario, however historically that was the first place of enforcing some level of accountability that:
 - the checklist were even looked at for a production service
 - the checklist may not be complete, but its status could be evaluated
 - there was some sort of historical record of that

I'm usually quite anti-extra levels of process. However in this case, we have been burnt many times by production services that are outside the bounds of current team knowledge and practices. Having some level of responsibility around this is important. I care not that it lies in a GitHub repository, but that it be trackable over time so that we can acknowledge that this information has been reviewed and prioritized by management and project teams.